src/content/guidelines/iconography/usage.mdx

```diff
@@ -29,6 +29,16 @@ tabs: ['Library', 'Usage', 'Contribute']
 
  </ClickableTile>
 </Column>
+<Column colLg="4" colMd="4"  noGutterSm>
```

Comment:
```suggestion
<Column offsetLg="4" colLg="4" colMd="4"  noGutterSm>
```

This tile needs the offset class to appear below the other two

Reply:
cool thanks, comitted ur suggestion! i was initially assuming it'd just be 3 flowing across but this is better + designer-approved now : )